salt/utils/cloud.py

```diff
@@ -2763,15 +2780,25 @@ def list_cache_nodes_full(opts=None, provider=None, base=None):
     return minions
 
 
-def cache_nodes_ip(opts, base=None):
+def get_cache_node(minion_id, base=None):
```

Comment:
Good catch!  The former function, cache_nodes_id, was never called anywhere in salt.  It appears that I replaced it with one that I was planning to use, get_cache_node, but abandoned when that design did not work out, and then forgot about.  So it is never called either.  So both versions are dead code.
  What would be the best path forward?
. keep both dead versions?
. keep my new dead version?
. keep the original dead version?
. kill both?